catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,317 @@
+
+    Contact [sm.fs.lcms@usda.gov](mailto:sm.fs.lcms@usda.gov) with any
+    questions or specific data requests.
+
+    * **Breiman, L., 2001.**
+    Random Forests. In Machine Learning. *Springer*,
+    45: 5-32
+    [doi:10.1023/A:1010933404324](https://doi.org/10.1023/A:1010933404324)
+
+    * **Chastain, R., Housman, I., Goldstein, J., Finco, M., and Tenneson, K., 2019.**
+    Empirical cross sensor comparison of Sentinel-2A and 2B MSI, Landsat-8 OLI, and Landsat-7 ETM
+    top of atmosphere spectral characteristics over the conterminous United States. In Remote
+    Sensing of Environment. *Science Direct*,
+    221: 274-285
+    [doi:10.1016/j.rse.2018.11.012](https://doi.org/10.1016/j.rse.2018.11.012)
+
+    * **Cohen, W. B., Yang, Z., Healey, S. P., Kennedy, R. E., and Gorelick, N., 2018.**
+    A LandTrendr multispectral ensemble for forest disturbance detection. In Remote Sensing of
+    Environment. *Science Direct*,
+    205: 131-140
+    [doi:10.1016/j.rse.2017.11.015](https://doi.org/10.1016/j.rse.2017.11.015)
+
+    * **Foga, S., Scaramuzza, P.L., Guo, S., Zhu, Z., Dilley, R.D., Beckmann,
+    T., Schmidt, G.L., Dwyer, J.L., Hughes, M.J., Laue, B., 2017.** Cloud
+    detection algorithm comparison and validation for operational Landsat data
+    products. In Remote Sensing of Environment.  *Science Direct*, 194: 379-390
+    [doi:10.1016/j.rse.2017.03.026](http://doi.org/10.1016/j.rse.2017.03.026)
+
+    * **U.S. Geological Survey, 2019.** USGS 3D Elevation Program Digital Elevation 
+    Model, accessed August 2022 at https://developers.google.com/earth-engine/datasets/catalog/USGS_3DEP_10m
+
+    * **Kennedy, R. E., Yang, Z., and Cohen, W. B., 2010.**
+    Detecting trends in forest disturbance and recovery using yearly Landsat time series: 1.
+    LandTrendr - Temporal segmentation algorithms. In Remote Sensing of Environment.
+    *Science Direct*,
+    114(12): 2897-2910
+    [doi:10.1016/j.rse.2010.07.008](https://doi.org/10.1016/j.rse.2010.07.008)
+
+    * **Kennedy, R., Yang, Z., Gorelick, N., Braaten, J., Cavalcante, L., Cohen, W., and Healey, S., 2018.**
+    Implementation of the LandTrendr Algorithm on Google Earth Engine. In Remote Sensing. *MDPI*,
+    10(5): 691
+    [doi:10.3390/rs10050691](https://doi.org/10.3390/rs10050691)
+
+    * **Sentinel-Hub, 2021.**
+    Sentinel 2 Cloud Detector. [Online].
+    Available at: [https://github.com/sentinel-hub/sentinel2-cloud-detector](https://github.com/sentinel-hub/sentinel2-cloud-detector)
+
+    * **Zhu, Z., and Woodcock, C. E., 2012.**. 
+    Object-based cloud and cloud shadow detection in Landsat imagery. In Remote Sensing of
+    Environment. *Science Direct*, 
+    118: 83-94
+    [doi:10.1016/j.rse.2011.10.028](https://doi.org/10.1016/j.rse.2011.10.028)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license('https://data.fs.usda.gov/geodata/rastergateway/treecanopycover/')
```

Comment:
I don't see data license information on that page. Is there a USFS- or USDA-wide page similar to https://science.data.nasa.gov/license/ or https://www.usgs.gov/faqs/what-are-terms-uselicensing-map-services-and-data-national-map

Reply:
There is a data disclaimer at the bottom of the page. Does this work? 

"Enterprise Data Disclaimer:

The USDA Forest Service makes no warranty, expressed or implied, including the warranties of merchantability and fitness for a particular purpose, nor assumes any legal liability or responsibility for the accuracy, reliability, completeness or utility of these geospatial data, or for the improper or incorrect use of these geospatial data. These geospatial data and related maps or graphics are not legal documents and are not intended to be used as such. The data and maps may not be used to determine title, ownership, legal descriptions or boundaries, legal jurisdiction, or restrictions that may be in place on either public or private land. Natural hazards may or may not be depicted on the data and maps, and land users should exercise due caution. The data are dynamic and may change over time. The user is responsible to verify the limitations of the geospatial data and to use the data accordingly."

Reply:
Not quite - it doesn't say anything about whether or when the data are free to use. Generally, federally-produced data have no restrictions, are in public domain , or carry the CC0 license (https://creativecommons.org/public-domain/cc0/), but finding a page that says so specifically would be great.

Reply:
Looks like we do not currently have a data-access page for TCC. I'm talking with our web developer to add in a data-access info to https://data.fs.usda.gov/geodata/rastergateway/treecanopycover/

This is what our data-access statement looks like for LCMS. Would something similar work for TCC? 
https://data.fs.usda.gov/geodata/rastergateway/LCMS/index.php#template-data-access

Reply:
I'm hoping our web dev can get to this in the next few days. Does that work to keep this pull request on hold for a few more days until the data-access statement is added to the webpage?

Reply:
If there's no urgency, I'd prefer to wait until the web page is updated.